main_test.go

```diff
@@ -112,6 +115,16 @@ type pkg struct {
 }
 
 func testPkgBackend(t *testing.T, lang, pycmd string, table pkg) {
+	tableMutex.Lock()
+	tempTable := pkg{
+		path: table.path,
+		lang: table.lang,
+		want: table.want,
+	}
+	tableMutex.Unlock()
+
+	table = tempTable
```

Comment:
I don't understand why copying a value that contains slices of stuff would cure the "concurrent map writes" error.
could you explain this ?
also, to be sure this cures that problem, you should re-add the 2 commits that introduced that feature :)

(but thanks for taking a stab at this!)

Reply:
Hmm, my apologies- perhaps I've fixed something that wasn't broken at the same time as those commits were reverted or something; I did have failing tests that stopped failing after doing this, I'm almost sure of it.

Let me dig deeper and get back to you- may as well can the PR, I daresay it's the barking up the wrong tree altogether!